assets/js/blocks/store-notices/style.scss

```diff
@@ -0,0 +1,8 @@
+.woocommerce-notice {
```

Comment:
I'm not 100% convinced about this class name. If I understand it properly, it's only used in the editor, right? Because in the frontend, the markup uses a different class. In that case, should we [prefix it with `.wc-block-`](https://github.com/woocommerce/woocommerce-blocks/blob/1db184daa15b348e281739d007ddfa3f74660f61/docs/contributors/contributing/coding-guidelines.md#naming)?

Reply:
Fixed 👍